### PR TITLE
Trivial changes in the Rust book.

### PR DESCRIPTION
**Pull request by ticki**
Just some small changes.

### CODE REVIEW

**Comment by Ticki** [comment]
Thank you, my fellow bot.

File: src/doc/trpl/unsafe.md [hunk 1]
```diff
@@ -100,10 +100,14 @@ that you normally can not do. Just three. Here they are:
 
 That’s it. It’s important that `unsafe` does not, for example, ‘turn off the
 borrow checker’. Adding `unsafe` to some random Rust code doesn’t change its
-semantics, it won’t just start accepting anything.
+semantics, it won’t just start accepting anything. But it will let you write
+things that _do_ break some of the rules.
 
-But it will let you write things that _do_ break some of the rules. Let’s go
-over these three abilities in order.
+You will also encounter the 'unsafe' keyword when writing bindings to foreign
```
Comment (steveklabnik) [inline src/doc/trpl/unsafe.md]: this should be in graves, not apostraphes, see line 101 and 102 above

**Comment by steveklabnik** [comment]
one tiny nit, otherwise r=me

**Comment by Ticki** [comment]
Done.

**Comment by steveklabnik** [comment]
Thank you!

**Comment by Ticki** [comment]
Yes, you bots are so friendly.

**Pull request merged**